<launch>
  <arg name="window_time" default="3" />
  <arg name="threshold" default="25" />

  <node name="avg_ultra_sonic" pkg="mb1222" type="avg_ultra_sonic.py" >
    <param name="window_time" value="$(arg window_time)" />
    <param name="threshold" value="$(arg threshold)" />
  </node>
</launch>